cg/services/analysis_starter/configurator/implementations/nextflow.py

```diff
@@ -0,0 +1,106 @@
+from abc import abstractmethod
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.apps.lims import LimsAPI
+from cg.constants import Workflow
+from cg.constants.nf_analysis import NextflowFileType
+from cg.services.analysis_starter.configurator.abstract_service import Configurator
+from cg.services.analysis_starter.configurator.file_creators.config_creator import (
+    NextflowConfigFileContentCreator,
+)
+from cg.services.analysis_starter.configurator.file_creators.utils import create_file, get_file_path
+from cg.services.analysis_starter.configurator.models.nextflow import NextflowCaseConfig
+from cg.store.models import Case
+from cg.store.store import Store
+
+
```

Comment:
I decided to create another layer of inheritance, as Nextflow pipelines have a big common functionality. The Nextflow pipeline configurations will inherit from this class